src/commands/postDeploy/checkLinuxWebAppDownDetector.ts

```diff
@@ -58,18 +57,32 @@ export async function checkLinuxWebAppDownDetector(correlationId: string, node:
                 return undefined;
             }
 
-            // tslint:disable-next-line: no-unsafe-any
-            detectorErrorMessage = await getLinuxDetectorError(context, linuxContainerStartFailureId, node, deployResultTime, node.root.client.fullName);
+            // time constraint of being less than the current time by 14.5 minutes is enforced system wide for all detectors
+            // however LinuxLogViewer detector ignores it and gets the most recent data
+            const nowTime: Date = new Date();
+            const startTimeDate: Date = new Date(nowTime.getTime() - (60 * 60 * 1000));
+            const endTimeDate: Date = new Date(nowTime.getTime() - (30 * 60 * 1000));
```

Comment:
I don't get this - is it saying the end time is 30 minutes ago? Before I deployed? Is this related to the above comment? If so - please explain how the 30/60 is related to 14.5

Reply:
I picked 60/30 because it's way beyond the 14.5 minute restraint that the detectors enforce.  I don't understand why they do it _(probably because Kusto query injection takes forever so you can't retrieve any detector information that isn't at least 14.5 minutes old)._

I could do start/end as 15/14.5, I guess but I didn't wanna mess with configuring the times since I asked Michimune and he said the LinuxLogViewer detector completely ignores that and just returns the most recent entry regardless.  My guess is because it's using app logs that populate while the deployment is happening, the time constraint doesn't apply to them but is a platform-wide constraint that they can't just ignore.